article-2/install_script.sh

```diff
@@ -9,17 +10,40 @@
 
 # SET LANG VARS
 
-echo 'export LC_ALL="en_US.UTF-8"'   >> ~/.bashrc
-echo 'export LANGUAGE="en_US:en"'    >> ~/.bashrc
+# BASE SOFT
+# wget http://debian.mirror.vu.lt/debian-sources.list && mv debian-sources.list /etc/apt/sources.list
+
+mv /etc/apt/sources.list /etc/apt/sources.list.backup
+echo "deb http://httpredir.debian.org/debian/ stable main contrib non-free
```

Comment:
Please, add a comment here. Something like:

"Here we add Debian sources to get packages from closest servers"